### ISSUE TITLE
[Bug]: gradle-plugin SNAPSHOT version not yet published

### ISSUE THREAD

**Issue by licaon-kter** [AndroidIDEOfficial/AndroidIDE#1620]
### What happened?
Source code: https://gitlab.com/fdroid/fdroidclient

### What version of AndroidIDE you're using?
F-Droid v2.7.0-beta (27003) 

### Relevant log output

```shell
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all files for configuration ':app:classpath'.
   > Cannot resolve external dependency com.itsaky.androidide:gradle-plugin:2.7.0-beta-9a9f65ca9-SNAPSHOT because no repositories are defined.
     Required by:
         project :app
> Plugin with id 'com.itsaky.androidide' not found.
```
...wants SNAPSHOT but that's not published: https://mvnrepository.com/artifact/com.itsaky.androidide/gradle-plugin

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
Thanks for the report. I'll fix it ASAP.

As a workaround, you can disable log sender from IDE Preferences -> Developer Options -> Enable LogSender. This will however result in application logs to be NOT shown in the IDE.

**Comment by itsaky** [comment]
Thanks for the report. I'll fix it ASAP.

**Comment by itsaky** [comment]
Update:
The issue seems to be occurring only for F-Droid builds. There was a configuration issue where the appropriate version name was not being used for the gradle plugin dependency for F-Droid builds. So, while the GitHub builds use the published versions, the F-Droid builds used the non-published, SNAPSHOT versions.

The above commit should fix the issue, but I still have to test those changes thoroughly.

**Comment by MrIkso** [comment]
Close, I'm tested latest dev building. Working

**Comment by itsaky** [comment]
Thanks for testing, @MrIkso.

I did test the changes on my own and can confirm that the issue has been resolved. There was another issue that I forgot to mention earlier which caused the build to fail when `apply plugin: '...'` syntax was used in Gradle build scripts. This issue has been resolved as well.

v2.7.1-beta will be released soon with the patch.

**Comment by itsaky** [comment]
Building the F-Droid client in AndroidIDE will require some changes to the build configuration.

For example, I see that the client uses Room database, which depends on the `sqlite-jdbc` library. `sqlite-jdbc` is known to have some issues when it is used in Android (see xerial/sqlite-jdbc#789, xerial/sqlite-jdbc#796 and #536). The issue was resolved in sqlite-jdbc v3.41.2.2 and Room v2.6.0-beta01.
